resources/authgear/templates/en/web/authflowv2/direct_access_disable.html

```diff
@@ -0,0 +1,28 @@
+{{ template "authflowv2/__direct_access_disable_page_frame.html" . }}
```

Comment:
The filename should be `direct_access_disabled.html`

Reply:
renamed